common/api/presentation-common.api.md

```diff
@@ -1086,6 +1086,17 @@ export interface HierarchyCompareOptions<TIModel, TNodeKey, TRulesetVariable = R
     resultSetSize?: number;
 }
 
+// @alpha
```

Comment:
What's the motivation for tagging as `alpha`, vs `internal` or `beta`?

Reply:
We've been adding multiple hierarchy level filtering related pieces over a long period of time and setting them all `@alpha`. The plan is to propagate all of them to `@beta` as soon as all these pieces are glued together and we enable filtering in our test app. I hope that's going to happen early next week.

Then, the plan is to integrate filtering into our consumer trees (Models, Categories, etc.) and finally propagate these APIs to `@public` with 4.0 release.

Reply:
I figured I didn't totally answer your question... The motivation is that we're not sure about the APIs until we make sure all of them can work together.

Reply:
Thanks.